packages/SidePanel/src/SidePanel.js

```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import PropTypes from "prop-types";
+import Dialog from "./components/Dialog";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import Overlay from "./components/Overlay";
+import Trigger from "./components/Trigger";
+import Group from "./components/Group";
+import FocusTrap from "./components/FocusTrap";
+import { extractChildren } from "./helpers";
+import useOffsetScroll from "./hooks/useOffsetScroll";
+import useBodyOverflow from "./hooks/useBodyOverflow";
+import useEscapeKey from "./hooks/useEscapeKey";
+
+const propTypes = {
+  /** The content for the SidePanel. */
+  children: PropTypes.node.isRequired,
+
+  /** The width of the open panel. */
+  width: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
```

Comment:
Seems like alphabetic order will be the easiest standard to enforce, even if it's not always as logical.

Reply:
Also, I think we should definitely use semantic enums for `width`.  I think we should impose a limited number of options to enforce consistency with our design system.

Reply:
I thought about that, but I think the component shouldn't enforce the width from this side. Would make more sense to provide a specific token that the developer can use and pass them to this prop. `<SidePanel width={tokens.sidepanel.width.half} />` as an example. What do you think? 

We can discuss this maybe in on our slack channel?

Reply:
We can merge and then modify this after we agree in a pattern?

Reply:
Agreed.  This shouldn't block the whole component.  I think it's important to follow up and decide on a pattern soon though.